ingestion/src/metadata/ingestion/ometa/mixins/lineage_mixin.py

```diff
@@ -156,31 +160,49 @@ def _create_lineage_by_table_name(
                 _get_formmated_table_name(str(from_table)),
             )
             from_entity: Table = self.get_by_name(entity=Table, fqdn=from_fqdn)
-
+            if not from_entity:
+                table_obj = self._separate_fqn(database=database, fqn=from_fqdn)
+                multiple_from_fqns = self.search_entities_using_es(
+                    service_name=service_name,
```

Comment:
if we have database available here we should pass that also that way you'll get more accurate hit.
so if you have database, databaseSchema and table pass all those to es
service: service_name AND database: database_name AND database_schema: database_schema AND name: table_name

when database is not available remove that and similarly when database_schema not available remove that one . that way you are trying to make sure to get a entity as close as possible